tools/packaging/common/envoy_bootstrap.json

```diff
@@ -736,57 +736,6 @@
       }
     }
   }
-  {{- else if .stackdriver }}
-  ,
-  "tracing": {
-    "http": {
-      "name": "envoy.tracers.opencensus",
-      "typed_config": {
-      "@type": "type.googleapis.com/envoy.config.trace.v3.OpenCensusConfig",
-      "stackdriver_exporter_enabled": true,
-      "stackdriver_project_id": "{{ .stackdriverProjectID }}",
-      {{ if .sts_port }}
```

Comment:
Do we only use sts_port for stackdriver? If so, can clean up that code in istio-agent as well in the future

cc @kyessenov 

(note: the STS I cleaned up was separate, we had 2 STS things in the agent)

Reply:
It's different impersonations I believe. Yeah, we can move it to Google agent as well.

Reply:
STS is also used for Google CA. STS clean up can be a separate work.

Reply:
Google CA's usage of STS is in https://github.com/istio/istio/pull/51043